Unified Cloud Navigation screenshot and style edits 2 (#207)
hashicorp/terraform-docs-common: website/docs/cloud-docs/registry/design.mdx

## Accessing the Configuration Designer

> Click the **Registry** button, and then click **<\> Design Configuration**.

Reviewer: This is a little unclear and also it said "button" :) 
```suggestion
Go to your organization's private registry, and then click **<\> Design Configuration**.
```

> The "Publish" page appears. Use the **Preview configuration** menu to review the generated code.

Reviewer: Forgive me. I had to :) 

```suggestion
The **Publish** page appears. Use the **Preview configuration** menu to review the generated code.
```
